libr/core/cmd_type.c

```diff
@@ -745,7 +745,10 @@ static int cmd_type(void *data, const char *input) {
 			r_core_cmd0 (core, "k anal/cc/*");
 			break;
 		case 0:
-			r_core_cmd0 (core, "k anal/cc/*~=cc[0]");
+			r_core_cmd0 (core, "tuc");
```

Comment:
we can probably simplify this  into 1 single line:

r_core_cmd0 (core, "tuc;tsc;ttc;tec");